### PR TITLE
Netsil v1.2.1

### PR DESCRIPTION
**Pull request by luhkevin**
Updating Netsil to version 1.2.1

### CODE REVIEW

**Comment by luhkevin** [comment]
Hey there,
Just wanted to say that we're ready to merge this whenever you guys are.

**Comment by ryadav88** [comment]
@luhkevin Can I also ask you to write short getting started docs for these packages so users don't have any issues installing and understanding how they work?

You can create a PR against https://github.com/dcos/examples and refer other examples listed there. These examples will finally go here: https://dcos.io/docs/1.8/usage/tutorials/.

This is also now part of the package submission guidelines: https://github.com/mesosphere/universe#submit-your-package

File: repo/packages/N/netsil/2/package.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "packagingVersion": "3.0",
+  "name": "netsil",
+  "version": "1.2.1",
+  "description": "Netsil is the monitoring and analytics product for API and microservices-driven applications.\n\nAn auto-discovered topology map of your services lets you find and fix failures faster.\n\nListen to your APIs, see your apps.",
+  "maintainer": "support@netsil.com",
+  "tags": ["netsil", "monitoring", "microservices"],
+  "preInstallNotes": "We recommend using 4 CPUs and 16 GB of memory for Netsil.\n\nYou will also need to install the `netsil-collectors` package, which feeds network and infrastructure data to Netsil for monitoring.\n\nAfter installation, go to `https://<dcos_url>/service/netsil` to access Netsil.\n\nYour dcos_url is the URL you initially used to access your DCOS cluster.",
```
Comment (ryadav88) [inline repo/packages/N/netsil/2/package.json]: Add this at the beginning of `preInstallNotes`: ```This DC/OS Service is currently in preview.```

Comment (ryadav88) [inline repo/packages/N/netsil/2/package.json]: Add the link to dcos/examples that you write at the end of `preInstallNotes`

File: repo/packages/N/netsil/2/package.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
```
Comment (ryadav88) [inline repo/packages/N/netsil/2/package.json]: Add `minDcosReleaseVersion` property

File: repo/packages/N/netsil-collectors/2/package.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
+{
```
Comment (ryadav88) [inline repo/packages/N/netsil-collectors/2/package.json]: Add `minDcosReleaseVersion `

File: repo/packages/N/netsil-collectors/2/package.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  "packagingVersion": "3.0",
+  "name": "netsil-collectors",
+  "version": "1.2.1",
+  "description": "The collectors package for Netsil.",
+  "maintainer": "support@netsil.com",
+  "tags": ["netsil", "monitoring", "microservices", "collectors"],
+  "preInstallNotes": "You may want to change the 'instances' field to control how many collectors you wish to install.\n\nFor instance, you may want one collector for every server you wish to monitor.\n\nIn addition, you may need to change the 'role' field to control on which nodes to install the collectors.",
```
Comment (ryadav88) [inline repo/packages/N/netsil-collectors/2/package.json]: Add this at the beginning of `preInstallNotes`: ```This DC/OS Service is currently in preview.```

Comment (ryadav88) [inline repo/packages/N/netsil-collectors/2/package.json]: Add dcos/examples link at the end of `preInstallNotes`

**Comment by luhkevin** [comment]
@ryadav88 Hi there Ravi, I've made the changes you've requested, and I've created a PR for a Netsil Usage guide in your dcos/examples repo.
https://github.com/dcos/examples/pull/149

Let me know if there's anything else you need before merging this one in.

**Pull request merged**